tracer/src/Datadog.Trace/AppSec/Rasp/MetaStructHelper.cs

```diff
@@ -92,4 +93,10 @@ public static byte[] ObjectToByteArray(object? value)
 
         return buffer;
     }
+
+    public static object ByteArrayToObject(byte[] value)
+    {
+        var formatterResolver = SpanFormatterResolver.Instance;
```

Comment:
I think you can use a default formatter here since you won't be using anything provided by `SpanFormatterResolver` (that is, you won't be serializing `Span` or `SpanModel`).

Reply:
In fact, `StandardResolver` is a composite that includes `PrimitiveObjectFormatter` and others, so you can probably use `StandardResolver` directly.

Reply:
Thanks for the info! I didn't know about the `StandardResolver` but it makes sense now.
Updated in 847f2a503199664c2c4a80bc2f375eb6a883e0eb